6-Typescript/server/handlers/todosController.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response } from 'express'
+import { Item } from '../interfaces/Item'
+import { DB } from '../interfaces/DB'
+export class TodoController {
+    db: DB
+    constructor(db: DB) {
+        this.db = db
+    }
+    getAllItems = (req: Request, res: Response<Item[] | string>) => {
+        this.db.getAllItemsFromDB(req.cookies.id)
+            .then((itemList: Item[]) => res.status(200).send(itemList))
+            .catch((err: Error) => res.status(400).send('could not get items, ' + err))
+    }
+    getItem = (req: Request, res: Response<Item | string>) => {
+        req.params?.id === '' ? res.status(400).send(`missing item id`) :
```

Comment:
u use `req.params.id` in a lot of places in this function. replace all places with one param.
ie.
`const id = req.params.id`
and then just use `id`

do the same in all of these functions. for example req.body.id etc..